src/Microsoft.DotNet.Interactive/KernelInfo.cs

```diff
@@ -52,8 +52,12 @@ public KernelInfo(string localName, string[]? aliases = null)
 
     private string CreateDisplayName()
     {
-        var displayName = $"{LanguageName} {LanguageVersion}".Trim();
-        return displayName;
+        if (string.IsNullOrWhiteSpace(LanguageName))
+        {
+            return LocalName;
+        }
+
+        return $"{LocalName} - {LanguageName}";
```

Comment:
The local name is always displayed in VS Code, so this will lead to the C# display name being `csharp - csharp - C# Script`

Reply:
Does seem to be the case: 
<img width="667" alt="image" src="https://user-images.githubusercontent.com/547415/213303236-ffec5199-39bf-431f-9089-36b057b67a38.png">

Reply:
Doesn't seem to be the case: 
<img width="667" alt="image" src="https://user-images.githubusercontent.com/547415/213303729-afd951c0-88e8-485f-930b-eb304bce04a8.png">